hack/test-go.sh

```diff
@@ -186,6 +186,7 @@ else
             -o -path "${kubernetes_path}/pkg/api/v1"                                                  \
             -o -path "${kubernetes_path}/test/e2e"                                                    \
             -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
+            -o -path "${kubernetes_path}/pkg/storage/etcd3"                                           \
```

Comment:
> Mind commenting on why are these disabled? Actually, each suite should be followed by such a comment.

Flaky upstream and we don't support etcd3.  Comment added and commit is marked drop.